cor_nonlinear/code/cor_nonlinear.R

```diff
@@ -1,55 +1,70 @@
 #-----------------------------------------------
 # obligatory to append to the top of each script
+# There is a bug on Windows that prevents renv from working properly. saved.system.libPaths provides a workaround:
+if (.Platform$OS.type == "windows") saved.system.libPaths=paste0(Sys.getenv ("R_HOME"), "/library")
 renv::activate(project = here::here(".."))
+if (.Platform$OS.type == "windows") {
+    options(renv.config.install.transactional = FALSE)
+    renv::restore(library=saved.system.libPaths, prompt=FALSE) # for a quick test, add: packages="backports"
+    .libPaths(c(saved.system.libPaths, .libPaths()))
+} else renv::restore(prompt=FALSE)
```

Comment:
@nhejazi What's the recommendation for this? `restore` or `activate`?

I've been doing manual `restore()`s from time to time.

Reply:
It would be better to avoid automatically calling `renv::restore()`